app.js

```diff
@@ -9,6 +9,7 @@ const boulderContainer = document.getElementById('boulder-container');
 
 const totalEl = document.getElementById('total');
 const winsEl = document.getElementById('wins');
+const lossesEl =document.getElementById('losses');
```

Comment:
This error is coming from eslint because you don't have a space after your equals sign on line 12.